smoke/LeafDevice/details/Details.cs

```diff
@@ -120,18 +120,7 @@ protected Task InitializeTrustedCertsAsync()
         {
             if (!string.IsNullOrEmpty(this.trustedCACertificateFileName))
             {
-                // Since Windows will pop up security warning when add certificate to current user store location;
-                // Therefore we will use CustomCertificateValidator instead.
-                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
did you test on Windows? I think the reason we need this logic here is because it is not possible to install CA certificate on Windows silently (without human interaction).  Therefore we use the callback approach.

Reply:
It will be great if we can automate to install cert on Windows. If so, you can remove these code.

Reply:
Quickstart Certs still fails for Windows. Which is odd because TransparentGateway test passes with workaround removed.